app/services/green_lanes/fetch_goods_nomenclature.rb

```diff
@@ -18,6 +18,8 @@ def call
           authorization: TradeTariffFrontend.green_lanes_api_token,
         },
       )
+
+      goods_nomenclature.get_declarable
```

Comment:
I guess this works fine for all the times we use this service?

Reply:
yes, if the goods is already declarable, it will be returned as it is:

```
  def get_declarable
    return self if declarable?

    descendants.find(&:declarable?)
end
```